test/system/160-volumes.bats

```diff
@@ -509,4 +509,14 @@ EOF
     is "$output" "" "Should print no output"
 }
 
+@test "podman ps -f" {
+    vol1=${PODMAN_TMPDIR}/v1_$(random_string)
+    mkdir $vol1
+    run_podman run  -d  --rm --volume $vol1:/$vol1 $IMAGE top
+    cid=$output
+    run_podman ps --noheading -f volume=${vol1} -q
+    is "$output" "$cid" "Should find container by volume"
+}
+
+
```

Comment:
Suggestion:
```bash
@test "podman ps -f" {
    vol1="/v1_$(random_string)"
    run_podman run -d --rm --volume ${PODMAN_TMPDIR}:$vol1 $IMAGE top
    cid=$output

    run_podman ps --noheading --no-trunc -q -f volume=$vol1
    is "$output" "$cid" "Should find container by volume"

    run_podman ps --noheading --no-trunc -q --filter volume=/NoSuchVolume
    is "$output" "" "ps --filter volume=/NoSuchVolume"

    # Clean up
    run_podman rm -f -t 0 -a
}
```

Reply:
Yeah it's more clean that my way of testing it